CribblyBackend/Controllers/GameController.cs

```diff
@@ -42,10 +44,12 @@ public async Task<IActionResult> Create([FromBody] Game game)
         {
             try
             {
+                Log.Information("Received request to create game: {@game}", game);
                 await gameService.Create(game);
             }
             catch (Exception e)
             {
+                Log.Information("Failed to create game: {@game}", game);
```

Comment:
Maybe we want to toss in `e.Message` for more info?

Reply:
Fixed. Thought Serilog would do that automatically, but I was wrong